### PR TITLE
Design fix of SGA setting page

### PR DESCRIPTION
**Pull request by amir-qayyum-arbisoft**
Design changes in sga settings page, added a settings tab and some style

design_settings

#70

### CODE REVIEW

File: edx_sga/static/css/edx_sga.css [hunk 1]
```diff
@@ -60,3 +60,69 @@
 .sga-block .error {
     color: red;
 }
+
+.wrapper-comp-settings .list-input.settings-list .field.comp-setting-entry {
```
Comment (carsongee) [inline edx_sga/static/css/edx_sga.css]: Please prefix all of these with `.modal-type-edx_sga` so we don't style other settings blocks on the same  page

File: edx_sga/templates/staff_graded_assignment/edit.html [hunk 2]
```diff
@@ -16,3 +16,7 @@
     {% endfor %}
   </ul>
 </div>
+
+<script type="application/javascript">
+ $("ul.action-modes").html(' <li class="view-button sga_editor_tab"><a href="#">{% trans "Settings" %}</a></li>')
```
Comment (carsongee) [inline edx_sga/templates/staff_graded_assignment/edit.html]: Since there is only one tab so far, you should pull out the `a` tag

**Comment by amir-qayyum-arbisoft** [comment]
@carsongee  done added .modal-type-edx_sga and remove a tag from list

**Comment by carsongee** [comment]
Great, looks good. Thanks again :+1:

**Pull request merged**